### DISCUSSION TITLE
Question about Proofreading Tracks

### DISCUSSION THREAD

**Discussion by cece3014** [talmolab/sleap#893] [Help!]
Hi Team, 
Super excited with SLEAP! it successfully has made predictions on unlabeled videos which is awesome! I am currently running inference, and correcting the tracks; however, I am slightly confused as to the extent of proofreading. 
Should tracks be demonstrated across all frames of the video? If this is the case, would that mean assigning tracks to each frame? or should SLEAP be able to do this after a few assigned tracks?

Thank you!

**Comment by roomrys** [comment]
Hi @cece3014,

Glad that your project is in full swing! Quick question, are you manually assigning tracks or are you using a Tracking Method in the inference pipeline?

![image](https://user-images.githubusercontent.com/38435167/182962578-8c950086-fc82-4a70-8105-5e29bcbfef78.png)
**Fig 1:** Tracker Method in Inference Pipeline

If you specify a tracker method, then SLEAP will produce tracks for you - usually too many tracks are created and some tracks are broken. You can specify how many instances to expect per frame using the [Cull to Target Instance Count](https://sleap.ai/develop/guides/proofreading.html#tracking-methods:~:text=identities%20with%20%E2%80%9C-,Cull%20to%20Target%20Instance%20Count,-%E2%80%9D.%20To%20use) option. After inference is run and SLEAP outputs the tracks (which will be added to the frames you predicted on), you will need to proofread to fix-up any broken tracks. Tracks will be added to all frames of the video if the "Predict On" target is the entire video.

Let us know if this helps, or if you need more clarification.

Thanks,
Liezl
